OpenRA.Mods.Cnc/Traits/Render/RenderShroudCircle.cs

```diff
@@ -19,37 +19,61 @@
 
 namespace OpenRA.Mods.Cnc.Traits
 {
-	class RenderShroudCircleInfo : TraitInfo<RenderShroudCircle>, IPlaceBuildingDecorationInfo
+	class RenderShroudCircleInfo : ITraitInfo, IPlaceBuildingDecorationInfo
 	{
+		[Desc("Color of the circle.")]
+		public readonly Color Color = Color.FromArgb(128, Color.Cyan);
+
+		[Desc("Contrast color of the circle.")]
+		public readonly Color ContrastColor = Color.FromArgb(96, Color.Black);
+
 		public IEnumerable<IRenderable> Render(WorldRenderer wr, World w, ActorInfo ai, WPos centerPosition)
 		{
 			var localRange = new RangeCircleRenderable(
 				centerPosition,
 				ai.TraitInfo<CreatesShroudInfo>().Range,
```

Comment:
This actually needs to be updated as well, else it crashes when more than 1 CreatesShroud trait is present.